### PR TITLE
Updated styleguide, filter_layout and system pages.

### PR DESCRIPTION
**Pull request by delrueba**

### CODE REVIEW

File: components/organisms/filter_layout.vue [hunk 1]
```diff
@@ -14,58 +14,58 @@
...
-              <input id="name" v-model="filter.name" type="text" name="name">
+          <div class="form-item">
+            <label for="name">Naam <span class="label-optional">(Optioneel)</span></label>
+            <input id="name" v-model="filter.name" type="text" name="name">
+          </div>
+          <checkbox-with-filter v-if="processors.length"
+                                :id="'services'"
+                                v-model="filter['processor[]']"
+                                :items="processors"
+                                :legend="'Verwerkende dienst'"
```
Comment (zero2one) [inline components/organisms/filter_layout.vue]: How will we translate this labels?

File: layouts/error.vue [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
-              <nuxt-link :to="'/'">homepage</nuxt-link></span>
-          </li>
-          <li>
-            <i class="icon-fancyback" aria-hidden="true" /><span>De terugknop van je browser gebruiken</span>
-          </li>
-        </ul>
+    <div class="help-block highlight">
+      <div class="highlight__inner">
+        <div>
+          <h2>Hoe kunnen we je verder helpen?</h2>
```
Comment (zero2one) [inline layouts/error.vue]: How will we translate these texts?

File: components/organisms/filter_layout.vue (hunk 1 above)
Comment (delrueba) [reply components/organisms/filter_layout.vue]: We won't. It's not a multi lingual site.

File: layouts/error.vue (hunk 2 above)
Comment (delrueba) [reply layouts/error.vue]: We won't. It's not a multi lingual site.

**Pull request merged**